content/sdk/10.js/00.ethers/00.index.md

```diff
@@ -17,16 +17,41 @@ Take a look at the [migration guide](/sdk/js/ethers/v6/migration) if you are mig
 
 ### Installation
 
-To install SDK for zkSync Era, run the command below in your terminal.
+To install SDK for zkSync Era, follow the process below:
```

Comment:
Could we clarify specifically which SDK they are installing in this page?

Reply:
I have added a callout to select either V5 and V6